book/website/community-handbook/style/style-figures.md

```diff
@@ -13,6 +13,14 @@ In this sub-chapter we have used the term **figure** to refer to an illustration
 The term **image** is used to refer to the file object itself.
 
 (ch-style-figures-licence)=
+
+## Images sources
+
+To facilitate figure modification, the availability of images sources is important to enable further editing.
+It is recommended to provide original source files along with the final format (such as `jpg` or `png`) used by figures.
+
+> **Example:** Photoshop uses source files with a `psd` extension which allows manipulation of image layers before an export into a format like `jpg`.
```

Comment:
At the risk of being pedantic, PSD is a file format, usually with a `.psd` extension.
We also might want to use a FOSS and/or open file format example like XCF?